controllers/teams.js

```diff
@@ -1036,7 +1036,7 @@ router.get('/:teamId/members', async (req, res, next) => {
 				user.userId.firstName || '',
 				user.userId.lastName || '',
 				roleTranslations[user.role.name],
-				user.userId.schoolId.name || '',
+				(user.userId.schoolId && user.userId.schoolId.name) || '',
```

Comment:
I actually had to look up how exactly this behaves. I think this might warrant a short comment, something like 
// returns school name if schoolid defined, empty string otherwise